src/components/stepforms/ColumnPicker.vue

```diff
@@ -41,6 +40,9 @@ export default class ColumnPicker extends Vue {
   @Prop({ default: null })
   dataPath!: string;
 
+  @Prop({ default: true })
+  sync!: boolean;
```

Comment:
Maybe rename to `syncWithSelectedColumn`? Anyhow, add a docstring to this attribute.